src/main/java/app/mapping/InstrumentDataRaw.java

```diff
@@ -6,30 +6,50 @@
 @Data
 public class InstrumentDataRaw {
     // Volume
-    double[] relVolume;
-    boolean[] absVolume;
+    public double[] relVolume;
+    public boolean[] absVolume;
 
     // Pitch
-    double[] pitch;
+    public double[] pitch;
 
     // Timbre
-    InstrumentEnum instrument;
+    public InstrumentEnum instrument;
 
     // Echo parameters
-    double[] delayEcho;
-    double[] feedbackEcho;
-    boolean[] onOffEcho;
+    public double[] delayEcho;
+    public double[] feedbackEcho;
+    public boolean[] onOffEcho;
 
     // Reverb parameters
-    double[] delayReverb;
-    double[] feedbackReverb;
-    boolean[] onOffReverb;
+    public double[] delayReverb;
+    public double[] feedbackReverb;
+    public boolean[] onOffReverb;
 
     // Filter parameters
-    double[] frequency;
-    boolean highPass;
-    boolean[] onOffFilter;
+    public double[] frequency;
+    public boolean highPass;
+    public boolean[] onOffFilter;
 
     // Panning
-    double[] pan;
+    public double[] pan;
+
+    public InstrumentDataRaw(double[] relVolume, boolean[] absVolume, double[] pitch, InstrumentEnum instrument,
```

Comment:
@ArtInLines why did you add a constructor here? Shouldn't Lombok provide exactly this constructor via @Data?